### ISSUE TITLE
Vermilion is blue again in secret lab trophies

### ISSUE THREAD

**Issue by Dav999-v** [TerryCavanagh/VVVVVV#862]
There used to be a bug where the trophies for time trial V ranks appeared as the wrong colors, for example, Vermilion and Victoria are switched around. This was fixed in 2.3.

A series of events (involving me accidentally corrupting my save files and resetting my trophies) led to the discovery that these colors are actually switched around again in 2.4.

For comparison, this is 2.3 (branch `2.3-updates`):
![2.3](https://user-images.githubusercontent.com/44736680/152862003-0e042b0c-14fe-4af7-822c-c63724737c57.png)

And this is the `master` branch:
![2.4](https://user-images.githubusercontent.com/44736680/152862057-76a8e8ff-e565-48aa-aa4d-b62820161bd7.png)

Bisecting landed me at this commit:
```
37b7615b71c3a2f44e03c47894383107850812ff is the first bad commit
commit 37b7615b71c3a2f44e03c47894383107850812ff
Author: Misa <infoteddy@infoteddy.info>
Date:   Thu Feb 18 22:20:11 2021 -0800

    Fix surface color masks
    
    This fixes the color ordering of every SDL_Surface in the game.
    
    Basically, images need to be loaded in ABGR format (except if they don't
    have alpha, then you use RGB? I'm not sure what's going on here), and
    then they will be converted to RGB/RGBA afterwards.
    
    Due to the surfaces actually being BGR/BGRA, the game used to use
    getRGBA/getRGB to swap the colors back around to BGRA/BGR, but I've
    fixed those too.

 desktop_version/src/Graphics.cpp          | 4 ++--
 desktop_version/src/GraphicsResources.cpp | 9 +++------
 desktop_version/src/Screen.cpp            | 7 ++-----
 3 files changed, 7 insertions(+), 13 deletions(-)

```
